src/qutip_tensorflow/core/data/__init__.py

```diff
@@ -7,6 +7,7 @@
 from .kron import *
 from .matmul import *
 from .mul import *
+from .expect import *
 from .expm import *
 from .project import *
-from .pow import *
+from .pow import *
```

Comment:
Minor nit, but can you fix your text editor so it correctly puts a newline as the last character of a file?

Reply:
I am manually using black to format the code. I thought it followed PEP 8 conventions (except for the number of characters for each line being 88 instead of 79). Quite shocked it does not do that here?

Reply:
Black should always put newlines at the ends of files, so something else must be going wrong.  Is black definitely running on everything?  It could also be something strange with your `git` option `autocrlf`, perhaps.

Reply:
I am running black locally for each file individually (i double checked that for the __init__ black removes the last new line). I will look closer to what the problem is. Since I will definitely have a merge conflict I will solve the last line issue there.